modules/tinymce/CHANGELOG.md

```diff
@@ -11,13 +11,16 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - The `tabindex` attribute is now copied from the target element to the iframe #TINY-8315
 - New `AutocompleterStart`, `AutocompleterUpdate` and `AutocompleterEnd` events #TINY-8279
 - New `mceAutocompleterClose`, `mceAutocompleterReload` commands #TINY-8279
+- New `FakeClipboard` API on the `tinymce` global #TINY-8353
 - New `slider` dialog component #TINY-8304
-- New `FakeClipboard` API to the `tinymce` global #TINY-8353
 - New `buttonType` property on dialog button components, supporting `toolbar` style in addition to `primary` and `secondary` #TINY-8304
+- New `imagepreview` dialog component, allowing preview and zoom of any image URL #TINY-8333
 
 ### Improved
 - The upload results returned from the `editor.uploadImages()` API now includes a `removed` flag, reflecting if the image was removed after a failed upload #TINY-7735
 - The dialog `redial` API will now only rerender the changed components instead of the whole dialog #TINY-8334
+- Dialog api `setData` method now uses a deep merge algorithm to support partial nested objects.
+- Dialog spec `initialData` type is now `Partial<T>` to match the underlying implementation details.
```

Comment:
```suggestion
- The dialog spec `initialData` type is now `Partial<T>` to match the underlying implementation details #TINY-8334
```

Reply:
So what you're saying is we should've picked up the lack of a changelog entry on that other PR 😂

Reply:
Yeah I wasn't too sure what to do about Jira numbers here... having something is better than nothing though 🤷